CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ x.x.x
 **BUG FIXES**
 - Fix `DirectoryService.DomainAddr` conversion to `ldap_uri` SSSD property when it contains multiples domain addresses.
 
+**CHANGES**
+- Execute SSH key creation alongside with the creation of HOME directory, i.e. 
+  during SSH login, when switching to another user and when executing a command as another user. 
```

Comment:
What about instead of an example cloud have the specific pam services: su ,ssh and sudo?

Is this a change or an enhancements?

Reply:
I see the difference a bit subtle in this case, but it could be considered more an enhancement.

Reply:
I agree with you